pydantic/main.py

```diff
@@ -301,13 +307,8 @@ class BaseModel(metaclass=ModelMetaclass):
         __signature__: 'Signature'
 
     Config = BaseConfig
-    __slots__ = ('__dict__', '__fields_set__')
-    # equivalent of inheriting from Representation
-    __repr_name__ = Representation.__repr_name__
-    __repr_str__ = Representation.__repr_str__
-    __pretty__ = Representation.__pretty__
-    __str__ = Representation.__str__
-    __repr__ = Representation.__repr__
+    __slots__ = ('__fields_set__',)
+    __doc__ = ''  # Null out the Representation docstring
```

Comment:
can we have a proper docstring, maybe as simple as "Base class for creating pydantic models.".

Also, can this be done via a proper docstring rather than `__doc__ = '..`?

Reply:
It's unclear that we want a real docstring here, since it gets inherited by subclasses. In particular, this line was added because tests were failing after classes started getting errant descriptions on them. Eg, [`test_schema`](http://github.com/FuegoFro/pydantic/blob/e67e1fe3a24aaea0a158557529711540a8061252/tests/test_dataclasses.py#L431) would need an added `'description': 'Base class for creating pydantic models.',` key, and that seems... incorrect?

I used `__doc__` rather than a normal docstring because it seemed clearer given that we're not setting a normal docstring, but rather removing one (which is a fairly unusual operation). If we did want to null out `Representation`'s docstring (for above reasons) without using `__doc__` we could do something like

```python
class BaseModel(Representation, metaclass=ModelMetaclass):
    """"""  # Empty docstring to null out the docstring from Representation
    ...
```
but I thought that might be more weird? Up to you though 🙂